src/apps/common/submodules/numbers/numbers.js

```diff
@@ -1072,8 +1058,11 @@ define(function(require) {
 			});
 		},
 
-		numbersPaintSpare: function(parent, dataNumbers, callback) {
+		numbersPaintSpare: function(args) {
```

Comment:
This function could be renamed to `numbersRenderSpare`, to follow the naming practices on other submodules for functions that display parts of the view.

Reply:
yep, sounds good!